generators/firechip/src/main/scala/BridgeBinders.scala

```diff
@@ -72,5 +78,6 @@ class WithDefaultFireSimBridges extends Config(
   new WithUARTBridge ++
   new WithBlockDeviceBridge ++
   new WithFASEDBridge ++
-  new WithTracerVBridge
+  new WithTracerVBridge ++
+  new WithTraceGenBridge
```

Comment:
I think it would probably be best in general to just add this to a TraceGen config but that's a nit.

Reply:
Fixed